### ISSUE TITLE
[Gutenberg WPCOM e2e tests i1] Tracking issue

### ISSUE THREAD

**Issue by fullofcaffeine** [Automattic/wp-calypso#46583]
**Project:** pc9Uba-8b-p2
**Previous tracking issue:** https://github.com/Automattic/wp-calypso/issues/45209
**Previous iteration/project:** pc9Uba-4N-p2

- [ ] https://github.com/Automattic/jetpack/issues/17642

**Comment by synora** [comment]
@ockham I noticed this issue has gone stale. It is still relevant or can it be closed?

**Comment by fullofcaffeine** [comment]
This is the tracking issue for this project: pc9Uba-8b-p2. Its ethos was to "move tests upstream" (from Calypso E2Es).

We wanted to do that by implementing a build process similar to what is currently working on Jetpack in repos that don't have it, and move all Gutenberg tests in Calypso, "upstream" to their "origin". We already incorporated that as a principle we follow as part of our platform work in Team Calypso (and previously Good Mountain). There are some A8C block repos that might still benefit from having CI like Jetpack, and we might still look into this later on a per-need basis.

We've also made a tangible improvement which was to improve the Jetpack build process by running its GB-related E2Es against the latest Gutenberg version on Github (pre-release or stable). This was done by @brbrr [here](https://github.com/Automattic/jetpack/pull/19132). 

I think we can close this one.
